model/model.go

```diff
@@ -120,6 +120,24 @@ type StorageAlias struct {
 	DeviceFile bool   `yaml:"devicefile,omitempty,flow"`
 }
 
+// InitializeDefaults ensure defaults are set such
+// as initializing pointer members
+func (si *SystemInstall) InitializeDefaults() {
```

Comment:
We need a rebase correct? I thought you added this as this based on previous PR

Reply:
This is from the previous commit. 

Only the three new commits are from this PR, but because I pushed the branch before the merge of the previous PR, it looks like 5 commits.

Reply:
There, I just force pushed and now you should only see the three new commits.